dispatcher/queue.go

```diff
@@ -1,54 +1,59 @@
 package dispatcher
 
 import (
-	"container/list"
+	"container/heap"
 	"sync"
 )
 
 // A PriorityQueue implements heap.Interface and holds Items.
 type PriorityQueue struct {
-	jobs *list.List
+	jobs []*Job
```

Comment:
IMO `[]*Job` should be it's own (unexported) type, and Len/Swap/Less/Push/Pop should be implemented on that type. 
Otherwise we end up with a confusing API for the PriorityQueue type, which has both Push/Enqueue and Pop/Dequeue

Reply:
Done in https://github.com/newscred/webhook-broker/pull/134/commits/c2ce21c6040245a0109ff1ad1d9111f5a0b0760d